packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -142,11 +151,27 @@ class OpPerfTelemetry {
 		// due to complexity of the different asynchronus scenarios of the op message lifecycle.
 		this.opLatencyLogger = createSampledLogger(logger);
 
+		const opsEventSampler: IEventSampler = (() => {
+			let eventCount = 0;
```

Comment:
This should probably start with -1 so that the first time it runs, an event is logged? At least that's what the `deltaLatencyEventSampler` above does.

Reply:
No point logging with just 1 op processed.